src/types.rs

```diff
@@ -474,8 +473,7 @@ pub struct RuntimeConfig {
 	#[serde(default)]
 	#[serde(with = "port_range_format")]
 	pub http_server_port: (u16, u16),
-	pub ipfs_seed_randomize: bool,
-	pub ipfs_seed: u64,
+	pub ipfs_seed: Option<u64>,
```

Comment:
Instead of comment below, we could put `///` comment here, which will be available as `cargo doc`:

E.g.
```
	/// Seed for IPFS keypair. If not set, or seed is 0, random seed is generated
	pub ipfs_seed: Option<u64>,
```